rust/toolchain.bzl

```diff
@@ -276,10 +281,10 @@ def _rust_toolchain_impl(ctx):
         clippy_driver = ctx.file.clippy_driver,
         target_json = ctx.file.target_json,
         target_flag_value = ctx.file.target_json.path if ctx.file.target_json else ctx.attr.target_triple,
-        rustc_lib = ctx.attr.rustc_lib,
+        rustc_lib = depset(ctx.files.rustc_lib),
         rustc_srcs = ctx.attr.rustc_srcs,
-        rust_std = rust_std,
-        rust_lib = rust_std,  # `rust_lib` is deprecated and only exists for legacy support.
+        rust_std = rust_std_files,
```

Comment:
I'm surprised that no test fails because it seems we didn't change this line, https://cs.opensource.google/bazel/rules_rust/+/main:rust/toolchain.bzl;l=190?q=toolchain.bzl, and `rust_std_files` now doesn't have `.label`

Reply:
Well, of couse it doesn't fail - that code is in toolchain.bzl, we have access to `ctx.attr.rust_std` there. Ignore this thread please :)